config/default.json

```diff
@@ -4,7 +4,7 @@
 	"elasticsearch": {
 		"host": "111.111.111.111:9200"
 	},
-	"port": 3000,
+	"port": 8080,
```

Comment:
Let's not change the default port number. If you want to change the port it runs on locally, you can set `PORT=8080` in your `.env` to [override the default](https://github.com/NYPL/discovery-api/blob/23a288b6e7bd6cb5a853a536420b3b83d5cf3848/app.js#L49)

Reply:
OK